### ISSUE TITLE
Tensor & Neural Network in PyTorch | Zerohertz

### ISSUE THREAD

**Issue by Zerohertz** [Zerohertz/zerohertz.github.io#90]
https://zerohertz.github.io/pytorch-tensor-nn/ 

 Import Packages1234>>> import torch>>> import numpy as np>>> import matplotlib.pyplot as plt>>> from torch.autograd import Variable  TensorScala (0-Dim Tensor)12345
